pom.xml

```diff
@@ -42,6 +42,7 @@
     <docker.tag.long>${docker.tag.prefix}9.${jetty9.minor.version}-${maven.build.timestamp}</docker.tag.long>
 
     <docker.openjdk.image>openjdk:8</docker.openjdk.image>
+    <gcloud-projectId-file>${project.build.directory}/gcloud-projectid.properties</gcloud-projectId-file>
```

Comment:
I am currently 4 levels deep in branches on branches on branches.  
I'm even having to merge separate branches together to get any progress.
This PR (the one that's now 23 days old) needs to be merged, so that the other pending PRs can be submitted.
Those other PRs cannot be submitted yet as they will (currently) include all of the changes for this PR as well.  That's why you only currently see 1 other PR.  This PR is a literal core PR for everything else.

Its either merge this, or drop it and just do all of the changes for all of the issues in a generic `testing` branch with no separate PRs.